apps/ta_dump.py

```diff
@@ -10,69 +10,83 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from matplotlib.backends.backend_pdf import PdfPages
+from scipy import stats
 
 import trgtools
 
-TICK_TO_SEC_SCALE = 512e-9 # s per tick
+TICK_TO_SEC_SCALE = 16e-9 # s per tick
 
-def window_length_hist(window_lengths):
+def window_length_hist(window_lengths, seconds=False):
     """
     Plot a histogram of the TA window lengths.
+
+    Optionally, use ticks or seconds for scaling.
     """
+    time_unit = 'Ticks'
+    if seconds:
+        window_lengths = window_lengths * TICK_TO_SEC_SCALE
+        time_unit = 's'
+
     plt.figure(figsize=(6,4))
-    plt.hist(np.array(window_lengths, dtype=np.uint64).flatten() * TICK_TO_SEC_SCALE, color='k')
+    plt.hist(window_lengths, color='k')
 
     plt.title("TA Time Window Length Histogram")
-    plt.xlabel("Time Window Length (s)")
+    plt.xlabel(f"Time Window Length ({time_unit})")
 
     plt.tight_layout()
-    plt.savefig("window_length_histogram.svg")
+    plt.savefig("ta_window_length_histogram.svg")
```

Comment:
Would be nice if the output folder was one of the configurables. Not for this PR though.